app/controllers/idv/review_controller.rb

```diff
@@ -85,7 +85,16 @@ def init_profile
       idv_session.create_profile_from_applicant_with_password(password)
       idv_session.cache_encrypted_pii(password)
       idv_session.complete_session
-      create_user_event(:account_verified) if idv_session.phone_confirmed?
+
+      if idv_session.phone_confirmed?
```

Comment:
return unless...

Reply:
I'm not committed to it, but my preference is to avoid returns towards the middle of functions